esphome/components/seeed_mr24hpc1/seeed_mr24hpc1.cpp

```diff
@@ -0,0 +1,1030 @@
+#include "esphome/core/log.h"
+#include "seeed_mr24hpc1.h"
+
+#include <utility>
+#ifdef USE_NUMBER
+#include "esphome/components/number/number.h"
+#endif
+#ifdef USE_SENSOR
+#include "esphome/components/sensor/sensor.h"
+#endif
+
+namespace esphome {
+namespace seeed_mr24hpc1 {
+
+static const char *const TAG = "seeed_mr24hpc1";
+
+// Prints the component's configuration data. dump_config() prints all of the component's configuration
+// items in an easy-to-read format, including the configuration key-value pairs.
+void MR24HPC1Component::dump_config() {
+  ESP_LOGCONFIG(TAG, "MR24HPC1:");
+#ifdef USE_TEXT_SENSOR
+  LOG_TEXT_SENSOR(" ", "HeartbeatTextSensor", this->heartbeat_state_text_sensor_);
```

Comment:
Please use spaces in the labels here
```suggestion
  LOG_TEXT_SENSOR(" ", "Heartbeat Text Sensor", this->heartbeat_state_text_sensor_);
```